portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetsFields.testcase

```diff
@@ -124,12 +139,48 @@ definition {
 	}
 
 	@description = "LPS-185229. Confirm that the user can change the field's order on Fields tab page."
-	@ignore = "true"
 	@priority = 4
 	test CanChangeFieldOrder {
+		task ("And clicks on plus button") {
+			LexiconEntry.gotoAdd();
+		}
 
-		// TODO LPS-185229 CanChangeFieldOrder pending implementation
+		task ("And checks id and label fields") {
+			Check.checkNotVisible(
+				checkboxName = "id",
+				locator1 = "Checkbox#ANY_CHECKBOX");
 
+			Check.checkNotVisible(
+				checkboxName = "label",
+				locator1 = "Checkbox#ANY_CHECKBOX");
```

Comment:
I would use the macro here too and in the other tests that make this check

Reply:
Oh yes, nice! Will work on that